### PR TITLE
XRSystem: Single page for devicechange event

### PR DESCRIPTION
**Pull request by Elchi3**
#### Summary

Have one page for the `device change` event of `XRSystem`. Redirect the `ondevicechange` event handler page to it.

#### Motivation

De-duplicated event pages.

### CODE REVIEW

File: files/en-us/web/api/xrsystem/devicechange_event/index.md [hunk 1]
```diff
@@ -18,36 +18,31 @@ browser-compat: api.XRSystem.devicechange_event
...
+
+ ```js
+ addEventListener("devicechange", (event) => { });
+
+ ondevicechange = (event) => { });
+ ```
+
+## Event type
+
+{{DOMxRef("Event")}}.
```
Comment (wbamberg) [inline files/en-us/web/api/xrsystem/devicechange_event/index.md:37]: FWIW I'm not very keen on this way of representing the event type. Is this an an H2 section that's only ever going to contain one word? I think I liked the table we had before better. Similarly, *if* we want to represent "cancelable" in a consistent way, isn't it better in a table than in prose?

Comment (Elchi3) [reply files/en-us/web/api/xrsystem/devicechange_event/index.md:37]: I was thinking of "Event type" analog to "Return value" in method pages, but maybe it should be an h3 under Syntax then?
I'm not yet sure about cancelable and if it is all that important. So far it has always been "no".

Comment (Elchi3) [reply files/en-us/web/api/xrsystem/devicechange_event/index.md:37]: I've started https://github.com/mdn/content/discussions/10544 so we can discuss the new event page type.

Comment (Elchi3) [reply files/en-us/web/api/xrsystem/devicechange_event/index.md]: I've added a commit that experiments with no syntax section but h3 headings for Trigger, Event type and Use cases.

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**